### PR TITLE
Add telemetry services to wallaby deployment

### PR DESCRIPTION
**Pull request by yadneshk**
Depends-On: https://github.com/openstack-k8s-operators/openstack-operator/pull/821

### CODE REVIEW

**Review by bshephar — APPROVED** [review]

File: devsetup/scripts/standalone.sh [hunk 1]
```diff
@@ -53,6 +53,7 @@ MANILA_ENABLED=${MANILA_ENABLED:-true}
 SWIFT_REPLICATED=${SWIFT_REPLICATED:-false}
 TLSE_ENABLED=${TLSE_ENABLED:-false}
 CLOUD_DOMAIN=${CLOUD_DOMAIN:-localdomain}
+TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
```
Comment (abays) [inline devsetup/scripts/standalone.sh:56]: This doesn't seem to otherwise be used in this file

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by karelyatin** [comment]
recheck Depends-On horizon fix

**Pull request merged**